members/M000303.yaml

```diff
@@ -4,6 +4,8 @@ contact_form:
   action: ""
   steps:
     - visit: "http://www.mccain.senate.gov/public/index.cfm/contact-form"
+    - wait:
+      - value: 10
```

Comment:
i would set this to something a bit lower unless a 10 second wait makes submissions succeed

Reply:
These changes are for Mccain's yaml. It's here as a mistake. Please see the other PR

Reply:
these still need to be removed from this PR